src/sections/Users/components/UsersListItem.js

```diff
@@ -0,0 +1,24 @@
+
+
+function UserListItem({ user }) {
+	let backgroundClass
+
+	if (user.gender === "male")
+	{
+		backgroundClass = "bg-blue"
+	}
+	else
+	{
+		backgroundClass = "bg-pink"
+	}
```

Comment:
This is where ternary operators can come in handy:

`const backgroundClass = user.gender === ' male' ? 'bg-blue' : 'bg-pink'`

This is essentially what you have but on a single line. Use is sparingly, usually a regular if statement is more readable, but this is a good use case for ternary

Reply:
I actually had that and changed it back for readability purposes. Next time I'll use a ternary instead. Thank you for the feedback